ledger-core/v2/network/consensus/gcpv2/phasebundle/ph3ctl/phase3.go

```diff
@@ -145,6 +148,9 @@ func (c *Phase3Controller) workerPhase3(ctx context.Context) {
 		defer runtime.UnlockOSThread()
 	}
 
+	phase3StartedAt := time.Now()
+	defer stats.Record(ctx, metrics.Phase3Time.M(float64(time.Since(phase3StartedAt).Nanoseconds())/1e6))
```

Comment:
this is incorrect as deferred call's arguments are evaluated immediately

Reply:
done